frontend/src/models/polls.ts

```diff
@@ -0,0 +1,28 @@
+import {User} from "./user";
+
+export enum PollState {
+    ACCEPTED = "ACCEPTED",
```

Comment:
Decide if it's either `PollState.ACCEPTED` or `PageType.News` (upper case or not).
TypeScript documentation only uppers the first letter